hivemq-edge/src/frontend/src/extensions/datahub/components/helpers/CheckPolicyReportToast.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC } from 'react'
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Button,
+  ButtonGroup,
+  Text,
+  VStack,
+} from '@chakra-ui/react'
+import { ProblemDetailsExtended } from '@/api/types/http-problem-details.ts'
+import { useTranslation } from 'react-i18next'
+
+interface CheckPolicyReportToastProps {
+  errors: ProblemDetailsExtended[]
+  onFitView?: (id: string) => void
```

Comment:
question: I'm a bit confused on the name as I do not understand what is the exact use of the function while reading the code. Could you please elaborate?

Reply:
NAme is indeed way too confusing. Renamed (`PolicyErrorReport`)